src/App.vue

```diff
@@ -401,6 +448,12 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
   async beforeDestroy(): Promise<void> {
     window.removeEventListener('localStorageUpdated', this.handleLocalStorageChange);
     window.removeEventListener('resize', this.setResponsiveClassDebounced);
+    if (this.prefersDarkScheme && this.handleThemeChange) {
+      this.prefersDarkScheme.removeEventListener('change', this.handleThemeChange);
```

Comment:
`this.prefersDarkScheme?.removeEventListener('change', this.handleThemeChange);`